### PR TITLE
Allow accordion collapse widgets to be clicked anywhere

### PR DESCRIPTION
**Pull request by brianhelba**
* Allow accordion collapse widgets to be clicked anywhere
 * The Bootstrap examples don't provide the optimal placement of the ``data-toggle`` property. It should be placed on the ``.panel-heading`` div, not the ``a`` element inside the ``.panel-title``.
* Convert some usage of Jade pipes into spans or block text continuations
 * In addition to being more maintainable, this actually slightly improves the spacing of the compiled HTML, making precise text selection in a browser easier.
 * Also, correct the spelling of "MongoDB".
* Remove inline nested elements from Jade templates
 * This syntax is less clear and more difficult to maintain.

### CODE REVIEW

**Comment by brianhelba** [comment]
@girder/developers I doubt this is the sort of thing we'd want to write a test for (because maybe there are legitimate exceptions), but we should remember to put the ``data-toggle`` on the ``.panel-heading`` in all future accordion collapse widgets.

**Comment by zachmullen** [comment]
What happens in this case for other elements embedded in panel headings that are clickable but not meant to toggle panel openness?

**Comment by brianhelba** [comment]
@zachmullen I'm not sure, but it seems easy enough to try out if we're interested. If the parent ``.panel-heading`` really does steal (or duplicate) all click events from a child element that has a higher ``z-index`` and its own click handler, then perhaps that's one of the "legitimate exceptions".

At the moment though, we don't have any such embedded elements in Girder, do we?

**Comment by zachmullen** [comment]
We do have some [in covalic](https://github.com/girder/covalic/blob/master/web_external/templates/body/scoringMetrics.jade#L33-L38).

**Comment by brianhelba** [comment]
@zachmullen So.....DTPRLGTY?

File: clients/web/src/templates/widgets/groupInviteDialog.jade [hunk 1]
```diff
@@ -39,17 +43,19 @@
...
+                    span Add as moderator
 
         if (level >= accessType.ADMIN)
           .panel.panel-default
-            .panel-heading: .panel-title
-              a(data-toggle="collapse", data-parent="#g-invite-role-container",
-                href="#g-invite-role-admin") Invite as administrator
+            .panel-heading(data-toggle="collapse",
+                           data-parent="#g-invite-role-container",
+                           data-target="#g-invite-role-moderator")
```
Comment (zachmullen) [inline clients/web/src/templates/widgets/groupInviteDialog.jade]: This should open the admin tab rather than the moderator tab.

**Comment by zachmullen** [comment]
Other than that one bug, this LGTM.

File: clients/web/src/templates/widgets/groupInviteDialog.jade (hunk 1 above)
Comment (brianhelba) [inline clients/web/src/templates/widgets/groupInviteDialog.jade]: Yikes, sorry.

**Comment by brianhelba** [comment]
Fixed that typo and rebased.

**Pull request merged**